packages/components/src/checkbox/src/Checkbox.module.css

```diff
@@ -0,0 +1,212 @@
+    --hop-Checkbox-box-background-color-checked: var(--hop-neutral-surface-active);
+    --hop-Checkbox-check-display-checked: block;
+    --hop-Checkbox-text-color-checked: var(--hop-neutral-text);
+    --hop-Checkbox-icon-color-checked: var(--hop-neutral-icon);
+
+    /* Disabled */
+    --hop-Checkbox-box-border-color-disabled: var(--hop-neutral-border-disabled);
+    --hop-Checkbox-box-background-color-disabled: var(--hop-neutral-surface-disabled);
+    --hop-Checkbox-box-border-color-checked-disabled: var(--hop-neutral-border-disabled);
+    --hop-Checkbox-box-background-color-checked-disabled: var(--hop-neutral-surface-disabled);
+    --hop-Checkbox-check-color-disabled: var(--hop-neutral-icon-disabled);
+    --hop-Checkbox-text-color-disabled: var(--hop-neutral-text-disabled);
+    --hop-Checkbox-icon-color-disabled: var(--hop-neutral-icon-disabled);
+
+    /* Focus */
+    --hop-Checkbox-box-border-color-focus: var(--hop-neutral-border-hover);
+    --hop-Checkbox-box-background-color-focus: var(--hop-neutral-surface-hover);
+    --hop-Checkbox-box-focus-ring-color: var(--hop-primary-border-focus);
+    --hop-Checkbox-box-border-color-checked-focus: var(--hop-neutral-border-active);
+    --hop-Checkbox-box-background-color-checked-focus: var(--hop-neutral-surface-active);
+    --hop-Checkbox-text-color-focus: var(--hop-neutral-text);
+    --hop-Checkbox-icon-color-focus: var(--hop-neutral-icon);
+
+    /* Invalid  */
+    --hop-Checkbox-box-border-color-invalid: var(--hop-danger-border-strong);
+    --hop-Checkbox-box-background-color-invalid: var(--hop-neutral-surface);
+    --hop-Checkbox-box-border-color-hover-invalid: var(--hop-danger-border-strong);
+    --hop-Checkbox-box-background-color-hover-invalid: var(--hop-danger-surface);
+    --hop-Checkbox-box-border-color-focus-invalid: var(--hop-danger-border-strong);
+    --hop-Checkbox-box-background-color-focus-invalid: var(--hop-danger-surface);
+    --hop-Checkbox-box-border-color-checked-invalid: var(--hop-danger-border-strong);
+    --hop-Checkbox-box-background-color-checked-invalid: var(--hop-danger-surface-strong);
+    --hop-Checkbox-check-color-invalid: var(--hop-danger-icon-strong);
+    --hop-Checkbox-text-color-invalid: var(--hop-danger-text-weak);
+    --hop-Checkbox-icon-color-invalid: var(--hop-danger-icon-weak);
+
+    /* Small */
+    --hop-Checkbox-box-sm-inline-size: 1rem;
+    --hop-Checkbox-text-sm-top-offset: calc((var(--box-inline-size) - (var(--hop-body-sm-line-height) * var(--hop-body-sm-font-size))) / 2); 
+    --hop-Checkbox-text-sm-margin-inline-start: calc(var(--hop-space-inline-sm) - var(--hop-space-inline-xs));
+
+    /* Medium */
+    --hop-Checkbox-box-md-inline-size: 1.5rem;
+    --hop-Checkbox-text-md-top-offset: calc((var(--box-inline-size) - (var(--hop-body-md-line-height) * var(--hop-body-md-font-size))) / 2); 
+    --hop-Checkbox-text-md-margin-inline-start: calc(var(--hop-space-inline-md) - var(--hop-space-inline-xs));
```

Comment:
this is because the gap is --hop-space-inline-xs, but we want extra space between the checkbox and text. We don't actually have a hopper token for that value so I put what the value should be, minus what's already there.